lib/internal/process/stdio.js

```diff
@@ -8,26 +8,64 @@ function setupStdio() {
   function getStdout() {
     if (stdout) return stdout;
     stdout = createWritableStdioStream(1);
+
+    Object.defineProperty(stdout._writableState, 'ended', {
+      set() {},
+      get() {
+        return false;
+      }
+    });
+    // XXX(Fishrock123): This should work I think, but it causes
+    // problems with some child-process related tests.
+    //
+    // Object.defineProperty(stdout, 'destroyed', {
+    //   set() {},
+    //   get() {
+    //     return true;
+    //   }
+    // });
```

Comment:
(The current patch works, but I think maybe having `destroyed = true` by default always would work better?) 9I don't think we can re-set it after `destroy()` and not hit what it is needed to skip.